bin/checkPad.js

```diff
@@ -5,29 +5,26 @@
 
 // As of v14, Node.js does not exit when there is an unhandled Promise rejection. Convert an
 // unhandled rejection into an uncaught exception, which does cause Node.js to exit.
-process.on('unhandledRejection', (err) => { throw err; });
 
-const npm = require('ep_etherpad-lite/node_modules/npm');
-const util = require('util');
+const process = require('process');
+const settings = require('ep_etherpad-lite/node/utils/Settings');
+settings.loglevel = 'debug';
 
+process.on('unhandledRejection', (err) => { throw err; });
```

Comment:
Move this back to line 8, above all of the `require()` statements (so that an unhandled Promise rejection in one of the includes will cause node to exit).